.travis.yml

```
@@ -1,8 +1,11 @@
-language: node_js
-node_js: 9
-sudo: required
-addons:
-  chrome: stable
-script:
-- npm run lint
-- npm run test
\ No newline at end of file
+language: node_js
+node_js: 9
+before_install:
+- curl -ks -u$ARTIFACTORY_USER:$ARTIFACTORY_PASS "https://d2lartifacts.artifactoryonline.com/d2lartifacts/api/npm/npm-local/auth/d2l" -o .npmrc
```

I think the 
```suggestion
- curl -ks -u$ARTIFACTORY_USER:$ARTIFACTORY_PASS "https://d2lartifacts.jfrog.io/d2lartifacts/api/npm/npm-local/auth/d2l" -o .npmrc
```
I think the artifactoryonline.com domain has been deprecated, and should be jfrog.io.